runtime/common/src/main/java/edu/snu/nemo/runtime/common/optimizer/pass/runtime/DataSkewRuntimePass.java

```diff
@@ -106,7 +106,8 @@ public PhysicalPlan apply(final PhysicalPlan originalPlan, final Map<String, Lis
         .map(list -> list.stream()
             .map(pair -> pair.left())
             .max(Integer::compareTo)
-            .orElseThrow(() -> new DynamicOptimizationException("Cannot find max hash value in a block.")))
+            .<DynamicOptimizationException>orElseThrow(
```

Comment:
I've done explicit typing here to avoid the JDK bug reported in
https://stackoverflow.com/a/40865318